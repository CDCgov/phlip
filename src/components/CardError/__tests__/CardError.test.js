import React from 'react'
import { shallow } from 'enzyme'
import { CardError } from '../index'

const props = {
  children: <div>I am a card error.</div>
}

describe('CardError component', () => {
  test('should render correctly', () => {
    expect(shallow(<CardError {...props} />)).toMatchSnapshot()
  })
})