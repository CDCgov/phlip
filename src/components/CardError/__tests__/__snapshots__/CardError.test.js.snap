// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardError component should render correctly 1`] = `
<Card
  style={
    Object {
      "display": "flex",
    }
  }
>
  <withStyles(GridWrapper)
    alignItems="center"
    container={true}
    justify="center"
    spacing={0}
    style={
      Object {
        "flex": "1",
      }
    }
  >
    <withStyles(GridWrapper)
      alignItems="center"
      container={true}
      direction="column"
      spacing={0}
    >
      <withStyles(GridWrapper)
        item={true}
      >
        <withTheme(Icon)
          color="#757575"
          size={175}
        >
          sentiment_very_dissatisfied
        </withTheme(Icon)>
      </withStyles(GridWrapper)>
      <withStyles(GridWrapper)
        item={true}
      >
        <withStyles(Typography)
          style={
            Object {
              "height": 300,
              "textAlign": "center",
              "width": 800,
            }
          }
          type="display2"
        >
          <div>
            I am a card error.
          </div>
        </withStyles(Typography)>
      </withStyles(GridWrapper)>
    </withStyles(GridWrapper)>
  </withStyles(GridWrapper)>
</Card>
`;
